### PR TITLE
Fix rooty water missing texture

### LINKED ISSUE #105: Rooty Water texture is missing when using the latest version of Dynamic Trees
Using [DynamicTrees-1.12.2-0.9.25](https://www.curseforge.com/minecraft/mc-mods/dynamictrees/files/3446235) (the version used in dev)

Using [DynamicTrees-1.12.2-0.9.29](https://www.curseforge.com/minecraft/mc-mods/dynamictrees/files/3812473) (curentelly latest for 1.12.2)

### PR DESCRIPTION
**Pull request by juraj-hrivnak**
Fixes #105 by removing the rooty water mixin, which Dynamic Trees has now built in.

### CODE REVIEW

File: src/main/java/com/fuzs/aquaacrobatics/core/ModCompatMixinHandler.java [hunk 1]
```diff
@@ -9,10 +9,6 @@
 @MixinLoader
 public class ModCompatMixinHandler {
     public ModCompatMixinHandler() {
-        AquaAcrobaticsCore.LOGGER.info("Aqua Acrobatics is loading mod compatibility mixins");
```
Comment (embeddedt) [inline src/main/java/com/fuzs/aquaacrobatics/core/ModCompatMixinHandler.java:12]: Please add the logging line back. I assume this was removed by accident.

**Review by embeddedt — COMMENTED** [review]
Looks good! I just have one request.

**Comment by juraj-hrivnak** [comment]
Ok, fixed! I must have overlooked it.

**Comment by embeddedt** [comment]
This is released in 1.15.2. Apologies for the delay in getting this out.

**Pull request merged**